packages/extensions/openapi-to-cadl/test/nh/cadl-output/models.cadl

```diff
@@ -170,39 +170,78 @@ model NotificationHubEntryModel {
 
 model NotificationHubContentModel {
   type?: NotificationHubContentModelType;
+
+  @friendlyName("notificationHubDescription")
   NotificationHubDescription?: NotificationHubDescriptionModel;
 }
 
 model NotificationHubDescriptionModel {
+  @friendlyName("xmlnsI")
   "xmlns:i"?: NotificationHubDescriptionModelXmlnsI;
+
```

Comment:
Based off of the offline discussion, I think the output here should look like:
```suggestion
  @projectedNames("json", "xmlns:i")
  "xmlnsI"?: NotificationHubDescriptionModelXmlnsI;
 
```
cc @timotheeguerin

Reply:
where the serialized name is in projected names